src/hooks/core/push-notification/usePushNotificationHook.ts

```diff
@@ -198,7 +201,7 @@ const usePushNotificationHook = () => {
               createdAt: message.created_at,
               updatedAt: message.created_at,
               rawJson: {},
-              attachmentJson: message.attachment,
+              attachmentJson: message.attachments,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `message.attachment` to `message.attachments` suggests that the message object structure has changed. Ensure that all parts of the codebase that interact with this object are updated accordingly to prevent potential issues.

```diff
- attachmentJson: message.attachment,
+ attachmentJson: message.attachments,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->